packages/esm-patient-chart-app/src/workspace/workspace-window.scss

```diff
@@ -92,7 +92,7 @@
     top: 0;
     left: 0;
     right: 0;
-    z-index: 1000;
+    z-index: 8000;
```

Comment:
I think we should adjust a competing z-index down, rather than just pushing this number very high, especially in a global override...

Reply:
According to what I've realized so far, the app bar was set to a z-index of `8000` allover. So this is what am trying to follow along with.  And for the case of this, the workspace header wasn't showing up 'cause it was blocked by the patient header

Reply:
Make it 8001 instead, I believe

Reply:
looks good on `8000`
![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/104269786/59f6692e-b74f-48c8-85dc-b067a2bf5203)